logprep/connector/elasticsearch/output.py

```diff
@@ -231,8 +237,7 @@ def store_custom(self, document: dict, target: str):
         """
         document["_index"] = target
         self._add_dates(document)
-        self.metrics.number_of_processed_events += 1
```

Comment:
this is needed because it was built in to check via the ratio from input to output events if the output using processors work. from the perspective of the output connector a SRE or pseudonym is an event.

Reply:
Thanks, I have re-added it.